PR#2941 – npm package: EJSONPARSE

```
Run npm clean-install
  npm clean-install
  shell: /usr/bin/bash -e {0}
npm error code EJSONPARSE
npm error JSON.parse Invalid package.json: JSONParseError: Expected ',' or '}' after property value in JSON at position 1202 (line 38 column 5) while parsing near "...ase\": \"^25.0.2\"\n    \"@semantic-release/c..."
npm error JSON.parse Failed to parse JSON data.
npm error JSON.parse Note: package.json must be actual JSON, not just JavaScript.
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2026-01-27T17_17_37_819Z-debug-0.log
Error: Process completed with exit code 1.
```

_Originally posted by @RalphHightower in https://github.com/RalphHightower/blog/issues/2941#issuecomment-3806471266_

--- a/package.json
+++ b/package.json
@@ -34,12 +34,12 @@
     "anti-trump"
   ],
   "devDependencies": {
-    "semantic-release": "^25.0.2"
+    "semantic-release": "^25.0.2",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^12.0.2",
     "@semantic-release/release-notes-generator": "^14.1.0",
-    "markdownlint-cli2": "^0.20.0",
+    "markdownlint-cli2": "^0.20.0"
   },
   "dependencies": {
     "npm": "^11.8.0"

